std/Lambda.hx

```diff
@@ -86,16 +84,20 @@ class Lambda {
 		Concatenate a list of iterables.
 		The order of elements is preserved.
 	**/
-	public static inline function flatten<A>( it : Iterable<Iterable<A>> ) : Array<A> {
-		return [for (e in it) for (x in e) x];
+	public static inline function flatten<A>(it:Iterable<Iterable<A>>):Array<A> {
+		return [
+			for (e in it)
+				for (x in e)
+					x
+		];
```

Comment:
This is too a lot more ugly than the original

Reply:
I think this is already fixed in the latest haxe-formatter version: https://github.com/HaxeCheckstyle/haxe-formatter/issues/498